model-engine/model_engine_server/inference/service_requests.py

```diff
@@ -48,7 +48,7 @@ def get_s3_client():
 
 def _read_function_to_network_endpoint_info():
     # Dictionary format: {servable_id: {remote: true/false, endpoint_type: "sync"/"async", destination: <str>},...}
-    # destination is either a celery queue name, i.e. llm_engine_server.<something>, or the full url for an http request,
+    # destination is either a celery queue name, i.e. launch.<something>, or the full url for an http request,
```

Comment:
update some comments?

Reply:
actually the celery queue name is still prefixed with `launch`